### DISCUSSION TITLE
Monorepo support for nx as well as Turborepo

### DISCUSSION THREAD

**Discussion by 0xdevalias** [shadcn-ui/ui#6150] [General]
In the docs for the new monorepo support:

- https://ui.shadcn.com/docs/monorepo

I see that `Turborepo` is supported:

> This will create a new monorepo project with two workspaces: web and ui, and [Turborepo](https://turbo.build/repo/docs) as the build system.

I was wondering if there are any plans to also support `nx`?

- https://nx.dev/

## See also

- https://github.com/shadcn-ui/ui/issues/778

**Comment by vgseven** [comment]
How to use shadcn for existing Turborepo ?

**Reply by Hayat4144** [reply]
I am also finding the way. How i we can add it in existing repo ? I installed in the packages/ui and try to compiled the packages in runtime . In **Next js15** transpilling doesn't work but in **version 14** it works.

**Comment by simon-schuster** [comment]
I'm looking forward to this! Nx support would be awesome!
